### ISSUE TITLE
Cost estimators crashes when used on an LQP containing a DummyTableNode

### ISSUE THREAD

**Issue by Felerius** [hyrise/hyrise#1500]
These two sqlite test statements crash when using #1473 because the subquery reformulation rule uses CostEstimatorLogical on the `(SELECT constant)` subqueries:
```sql
SELECT a FROM id_int_int_int_100 WHERE a IN (SELECT 14) AND b > (SELECT 15)
```
```sql
SELECT a FROM id_int_int_int_100 WHERE a IN (SELECT 11) AND b > (SELECT 11)
```
The crash happens because DummyTableNode does not override `derive_statistics_from` and the default implementation requires a left input.

**Comment by mrzzzrm** [comment]
Do you need a fix for this, or did you just happen to notice this and it doesn't bother you further? In the latter case, I'd fix this with #1489 - which will probably still take a bit until merged.

**Comment by Felerius** [comment]
Is there an easy way to create TableStatistics for a single row table so that we could fix it for now? Otherwise we could just disable these tests for the PR.

**Comment by janetzki** [comment]
Fixed by #1489
